src/pretix/presale/templates/pretixpresale/event/fragment_product_list.html

```diff
@@ -5,20 +5,28 @@
 {% load thumb %}
 {% load eventsignal %}
 {% load rich_text %}
-{% for tup in items_by_category %}
-    {% if tup.0 %}
-        <section aria-labelledby="category-{{ tup.0.id }}"{% if tup.0.description %} aria-describedby="category-info-{{ tup.0.id }}"{% endif %}>
-            <h3 id="category-{{ tup.0.id }}">{{ tup.0.name }}</h3>
-            {% if tup.0.description %}
-                <div id="category-info-{{ tup.0.id }}">{{ tup.0.description|localize|rich_text }}</div>
+{% for tup in items_by_category %}{% with category=tup.0 items=tup.1 form_prefix=tup.2 %}
+    {% if category %}
+        <section aria-labelledby="{{ form_prefix }}category-{{ category.id }}"{% if category.description %} aria-describedby="{{ form_prefix }}category-info-{{ category.id }}"{% endif %}>
+            <h3 id="{{ form_prefix }}category-{{ category.id }}">{{ category.name }}
+                {% if category.category_has_discount %}
```

Comment:
If we add the subevent-property to the DummyCategory, then this would be the HTML.
```suggestion
                {% if category.subevent %}
                    &nbsp; <small><span class='fa fa-calendar'></span> {{ category.subevent }}</small>
                {% endif %}
                {% if category.category_has_discount %}
                    &nbsp;
```
![Bildschirmfoto 2024-10-11 um 13 10 40](https://github.com/user-attachments/assets/a3fccec8-eb46-43d6-ae14-bd4870be04fb)